src/Native/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -834,7 +834,14 @@ bool QueryCacheSize()
 bool QueryLogicalProcessorCount()
 {
 #if HAVE_SYSCONF
-    g_cLogicalCpus = sysconf(_SC_NPROCESSORS_CONF);
+    int sysConfName;
+#if defined(_TARGET_WASM_)
```

Comment:
Nit: The other places in this file use `_WASM_`, not `_TARGET_WASM_`.

Reply:
Fixed